Gemfile

```diff
@@ -23,6 +23,10 @@ end
 
 gem 'simplecov', '~> 0.8'
 
+if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
+  gem 'json', '< 2.0.0'
+end
```

Comment:
It gets installed as a dependency of our build as a whole, and then errors on older rubies. I think this is related to our cross repo testing.